### ISSUE TITLE
non-interactive usage ?

### ISSUE THREAD

**Issue by garyng2000** [ethereum/staking-deposit-cli#250]
is there a way to use this in non-interactive way especially the 'existing-mnemonic' command ? I see there is supposed to be options but then there are 'confirmation' all over the place even if they have been provided.

**Comment by garyng2000** [comment]
what is the procedure to build the single file binary ? I have successfully changed the python code to not doing the prompt thing but don't know how to produce a single file binary

**Comment by garyng2000** [comment]
nvm, found it

**Comment by wackerow** [comment]
@garyng2000 Sorry for delay, but sounds like you figured this out. If you're all set here, please go ahead and close this issue. Thanks!

**Comment by VVander** [comment]
@wackerow I don't think this issue is resolved. I noticed today as I was setting up a dockerfile using the most recent release (v2.7.0) that the `--non_interactive` flag doesn't seem to do much. When making a new mnemonic, it still prompts the user to select a language, for example, unless you supply the `--language` flag. Unfortunately, not all prompts have a flag for me to supply; e.g. there is no way to prevent a prompt for a passphrase.

@garyng2000 if you still have the edits locally, would you mind making a PR?

**Comment by garyng2000** [comment]
pull from my repo, it is tracking this
